packages/action-bar/src/ActionBar.ts

```diff
@@ -97,7 +101,7 @@ export class ActionBar extends FocusVisiblePolyfillMixin(SpectrumElement) {
 
     public override render(): TemplateResult {
         return html`
-            <sp-popover ?open=${this.open} id="popover">
+            <${Popover.tag} ?open=${this.open} id="popover">
```

Comment:
Love this! We used to do something similar at Red Hat at one point though we used the tag attribute for the CMS rather than internal component referencing.